src/components/BatchSearchForm.vue

```diff
@@ -32,28 +36,28 @@
               {{ $t('batchSearch.learnMore') }}
             </a>
           </p>
-          <b-form-group label-size="sm" :label="$t('batchSearch.description')">
-            <b-form-textarea v-model="description" rows="2" max-rows="6"></b-form-textarea>
-          </b-form-group>
-          <b-form-group v-if="isServer" label-size="sm" :label="`${$t('batchSearch.projects')} *`">
-            <div class="batch-search-form__projects container p-0">
-              <b-form-checkbox-group v-model="projects" stacked>
-                <b-form-checkbox
-                  v-for="project in availableProjects"
-                  :key="project"
-                  :value="project"
-                  :disabled="isProjectDisabled(project)"
-                >
-                  {{ project | startCase }}
-                </b-form-checkbox>
-              </b-form-checkbox-group>
-            </div>
+          <b-form-group :label="$t('batchSearch.description')" label-size="sm" class="batch-search-form__description">
+            <b-form-textarea v-model="description" max-rows="6" rows="2"></b-form-textarea>
           </b-form-group>
           <b-form-group
             v-if="isServer"
+            :label="`${$t('batchSearch.projects')} *`"
+            class="batch-search-form__projects"
             label-size="sm"
+          >
+            <multiselect
+              v-model="projects"
+              :allow-empty="false"
+              :multiple="true"
+              :options="availableProjects"
+              :searchable="true"
```

Comment:
The props `:multiple="true"` and `:searchable="true"` can be set without `="true"`. Also `searchable` is default to true so we can remove it.

Reply:
:broom: